pom.xml

```
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-codex-ekb</artifactId>
-  <version>1.9.2-SNAPSHOT</version>
+  <version>2.0.0-SNAPSHOT</version>
```

```suggestion
  <version>1.10.0-SNAPSHOT</version>
```
Major version increasing only if breaking changes

---

The ticket says v2.0.0 [MODCXEKB-131](https://issues.folio.org/browse/MODCXEKB-131)
But ok, will change it